sktime/transformations/series/detrend/_deseasonalize.py

```diff
@@ -379,15 +382,14 @@ class STLTransformer(_SeriesToSeriesTransformer):
         than 1, the LOESS is used every low_pass_jump points and values between
         the two are linearly interpolated. Higher values reduce estimation
         time.
-
-    Attributes
-    ----------
-    trend_ : pd.Series
-        Trend component.
-    seasonal_ : pd.Series
-        Seasonal components.
-    resid_ : pd.Series
-        Residuals component.
```

Comment:
why did you remove Attributes section?

Reply:
because the attributes got removed.
The design switched to returning them as the `transform` output.